src/RelativeTime/RelativeTime.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import styled from 'styled-components'
+import {RelativeTimeElement} from '@github/time-elements'
+import {createComponent} from '@lit-labs/react'
+import {ComponentProps} from '../utils/types'
+import sx from '../sx'
+
+const RelativeTime = styled(createComponent(React, 'relative-time', RelativeTimeElement))(sx)
```

Comment:
types by default here don't seem to be correct (at least when imported into another project)

Reply:
or, at least they don't support `Date` objects, which may be expected?

Reply:
https://github.com/github/memex/pull/13207#discussion_r1017906919

Reply:
Types are correct. I followed up in that thread:

> Thanks for the catch! The way the properties work, datetime can only ever be a string, but date can be set to a Date (overriding datetime).